src/less/apps.less

```diff
@@ -95,6 +95,58 @@ img.icon-enricher {
 /* ----------------------------------------- */
 
 .sfrpg {
+
+    // Bubbles
+    .bubble {
+        margin: 0.5em 0.5em 1em 0.5em;
```

Comment:
I was playing around with the margins and I find `0.1em 0.1em 0.5em` nicer as it has less dead space. Lmk what you think

Reply:
Works for me, although I think I prefer the 0.5em on top of the bubbles rather than below, just to give a little extra space between the subtabs and the first bubble.
I'll switch it to 0.5em 0.1em 0.1em 0.1em